tinytuya/Cloud.py

```diff
@@ -466,6 +466,7 @@ def getconnectstatus(self, deviceid=None):
     def getdevicelog(self, deviceid=None, start=None, end=None, evtype=None, size=0, max_fetches=50, start_row_key=None, params=None):
         """
         Get the logs for a device.
+        https://developer.tuya.com/en/docs/cloud/0a30fc557f?id=Ka7kjybdo0jse

```

Comment:
I both love including this and worry that they will change the URL over time. 🤷  Worth keeping.

Reply:
I just figured you could always pull it out of the Internet Archive should it disappear.  https://web.archive.org/web/20230225125428/https://developer.tuya.com/en/docs/cloud/0a30fc557f?id=Ka7kjybdo0jse